src/analysis/sampling/plotSampleHist.m

```diff
@@ -119,49 +150,68 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
     currLB = 1e6;
     currUB = -1e6;
     for i = 1:length(samples)
-        if commonModelFlag
-            id = findRxnIDs(models, rxnNameList{j});
-        else
-            id = findRxnIDs(models{i}, rxnNameList{j});
-        end
-        if isempty(id)
-            if commonModelFlag
-                id = findRxnIDs(models, [rxnNameList{j} '_r']);
-            else
-                id = findRxnIDs(models{i}, [rxnNameList{j} '_r']);
-            end
-            if isempty(id)
-                warning('Reaction does not exist');
-            end
-        end
-        currLB = min(currLB, min(samples{i}(id,:)'));
-        currUB = max(currUB, max(samples{i}(id,:)'));
+        id = findRxnIDs(model, rxnNames{j});
```

Comment:
This no longer works with independent models (i.e. if you compare different models which don't have the same rxn order.